.github/workflows/superset-python-integrationtest.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.11"]
```

Comment:
I think we still want this to be running on the lowest version of Python we support.

Reply:
It's temporal Py3.11 CI for testing purposes.
I will revert the first commit later.